app/Amortization/Amortization.php

```diff
@@ -48,7 +48,7 @@ public function repaymentAmountSuperLoan(float $percentage = 0.0)
     {
         $amount = floor((($percentage  / 100) *  $this->order->repayment) / 100) * 100;
         if (RepaymentCycle::find($this->order->repayment_cycle_id)->name == RepaymentCycle::CUSTOM) {
```

Comment:
the if and else is returning same thing `$amount`

Reply:
resolved